examples/tests/testutils.py

```diff
@@ -4,40 +4,18 @@
 
 import re
 import ast
-import sys
 from pathlib import Path
-import subprocess
 
+import pygfx as gfx
 
 ROOT = Path(__file__).parents[2]  # repo root
 examples_dir = ROOT / "examples"
 screenshots_dir = examples_dir / "screenshots"
 diffs_dir = screenshots_dir / "diffs"
 
-
-def get_wgpu_backend():
-    """
-    Query the configured wgpu backend driver.
-    """
-    code = "import wgpu.utils; info = wgpu.utils.get_default_device().adapter.info; print(info['adapter_type'], info['backend_type'])"
```

Comment:
I'm happy if you want to make a change to wgpu that would allow an environment variable to override the default choice, but generally speaking it seemed weird to me that given the state of PyGFX today (more mature than when this test was likely written) you were testing the default from wgpu instead of what `class Shared` returned.

Reply:
Good point. It was only recently that pygfx had the option to use another adapter than the default.

This query was done in a subprocess, otherwise this module (i.e. the tests) would error in envs that do not have a wgpu adapter.

Reply:
do you have a way to if the new implementation can error?

Reply:
Looks like all our builds are on Linux and have lavapipe enabled. So it wont be triggered. So I guess the change is fine. We can do the subprocess thing if/when we add builds that don't have an adapter.

Reply:
ok see https://github.com/pygfx/pygfx/pull/827/files#r1688066345